src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -317,6 +317,10 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
 
         // need to load the pending item
         this.fetchFeedItems();
+
+        if (onTaskAssignmentUpdate) {
```

Comment:
Should we assign a `defaultProps` value of `noop` for this prop, similar to `onTaskUpdate`?

Reply:
Seems, there's already a default prop of `noop` :-) 
https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/ActivitySidebar.js#L116